src/NzbDrone.Core/Download/Clients/DownloadStation/DownloadStation.cs

```diff
@@ -0,0 +1,302 @@
+                    Status = GetTorrentStatus(torrent.Status),
+                    Message = GetMessage(torrent)                    
+                };
+
+                items.Add(item);
+            }
+
+            return items;
+        }
+
+        private string GetMessage(DownloadStationTask torrent)
+        {
+            if (torrent.StatusExtra != null)
+            {
+                switch (torrent.Status)
+                {
+                    case DownloadStationTaskStatus.Error:
+                        return torrent.StatusExtra["error_detail"];
+                    case DownloadStationTaskStatus.Extracting:
+                        return torrent.StatusExtra["unzip_progress"];
+                }
+            }
+
+            return null;
+        }
+
+        private DownloadItemStatus GetTorrentStatus(DownloadStationTaskStatus status)
+        {
+            if (StatusTable.ContainsKey(status))
+            {
+                return StatusTable[status];
+            }
+
+            return DownloadItemStatus.Downloading;
+        }
+
+        public override DownloadClientStatus GetStatus()
+        {
+            var data = _proxy.GetConfig(Settings);
+            
+            try
+            {
+                var path = data["default_destination"] as string;
+
+                if (Settings.TvCategory.IsNotNullOrWhiteSpace())
+                {
+                    path = string.Format("{0}/.{1}", path, Settings.TvCategory);
+                }            
+
+                return new DownloadClientStatus
+                {
+                    IsLocalhost = Settings.Host == "127.0.0.1" || Settings.Host == "localhost",
+                    OutputRootFolders = new List<OsPath> { _remotePathMappingService.RemapRemoteToLocal(Settings.Host, new OsPath(path)) }
+                };
+            }
+            catch (DownloadClientException)
+            {
+                _logger.Debug("Failed to get config from Download Station");
+
+                throw;
+            }
+
+            throw new NotImplementedException();
+        }
+
+        public override void RemoveItem(string downloadId, bool deleteData)
+        {
+            _proxy.RemoveItem(downloadId, deleteData, Settings);            
+        }
+
+        protected override string AddFromMagnetLink(RemoteEpisode remoteEpisode, string hash, string magnetLink)
+        {
+            _proxy.AddTorrentFromUrl(magnetLink, GetDownloadDirectory(), Settings);
+
+            var items = _proxy.GetItems(Settings).Where(t => t.Additional.Detail["uri"] == magnetLink);
+
+            if (items.Any())
+            {
+                try
+                {
+                    return items.Single().Id;
+                }
+                catch (InvalidOperationException)
+                {
+                    var ids = string.Join(",", items.Select(t => t.Id));
+                    _logger.Debug(string.Format("Multiple {0} tasks in Download Station: {1}", magnetLink, ids));
```

Comment:
Don't use `string.Format` within the logger, `_logger.Debug("Multiple {0} tasks in Download Station: {1}", magnetLink, ids);` will work.